packages/stores/src/providers/WebsiteStatusProvider.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useSubscription } from '@deriv/api';
+import useStore from '../useStore';
+import { useWS } from '@deriv/shared';
+import merge from 'lodash.merge';
+
+const WebsiteStatusProvider = ({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, is_subscribed, subscribe, unsubscribe } = useSubscription('website_status');
+    const {
+        client: { is_logged_in, is_logging_in, loginid },
+        website_status: { update },
+    } = useStore();
+    const WS = useWS();
```

Comment:
@adrienne-deriv Please don't use the `useWS` directly, For calling the server we should only go through `useSubscription`, `useFetch`, and `useRequest` hooks 🙏🏻